nomad/vault_noop.go

```diff
@@ -44,12 +52,18 @@ func (v *NoopVault) LookupToken(_ context.Context, _ string) (*vapi.Secret, erro
 	return nil, errors.New("Vault client not able to lookup tokens")
 }
 
-func (v *NoopVault) RevokeTokens(_ context.Context, _ []*structs.VaultAccessor, _ bool) error {
-	return errors.New("Vault client not able to revoke tokens")
+func (v *NoopVault) RevokeTokens(_ context.Context, tokens []*structs.VaultAccessor, _ bool) error {
+	for _, t := range tokens {
+		v.logger.Warn("Vault token is no longer used, but Nomad is not able to revoke it. The token may need to be revoked manually or will expire once its TTL reaches zero.", "accessor", t.Accessor, "ttl", t.CreationTTL)
```

Comment:
I'm not sure about these log messages. Not sure about their level (`WARN`), the actual text, or if they should even be emitted in the first place 😅

Reply:
The text you've got here is great. But yeah, maybe `DEBUG` level? It's only actionable inasmuch as some cluster operators will want to go clean these up manually, but they don't _need_ to.

Reply:
Dropped to `DEBUG`.